priv/repo/migrations/20180220231528_create_speakers.exs

```diff
@@ -7,7 +7,6 @@ defmodule CbusElixir.Repo.Migrations.CreateSpeakers do
       add :url, :string
       add :title, :string
       add :email, :string
-      add :meeting_id, :integer
```

Comment:
wat - is this like Rails, where editing an already-run migration is Bad Mojo?

Reply:
I know. I went ahead and edited it since we just have the two tables. Not great to do. Darby and I talked about that yesterday while pairing. We made an exception at that time and I did it now.

Reply:
But it's pushed and merged now...

Reply:
Okay. I can go back and change it.